client/lib/analytics/index.js

```diff
@@ -522,6 +522,23 @@ const analytics = {
 	},
 };
 
+/**
+ * Loading Google analytics independently from the rest of the tracking scripts.
+ *
+ * Why? Because ad-tracking and google-analytics have two different switches and we
+ * would probably not want one to stop the other.
+ *
+ * Moreover, analytics gets loaded with the page load, while the tracking is lazy-loaded
+ * during actions.
+ */
+if ( isGoogleAnalyticsAllowed() ) {
+	try {
+		loadScript( 'https://www.google-analytics.com/analytics.js' );
+	} catch ( error ) {
+		debug( 'GA script failed to load properly: ', error );
+	}
+}
+
```

Comment:
Yes this is a much better way +1

Initially had some concerns that we might potentially end up using the `window.ga` object before it's ready but looks like that part is taken care already by this:

https://github.com/Automattic/wp-calypso/blob/master/client/lib/analytics/index.js#L40

So it's all good it seems.

Maybe we could move this bit of code up there just so it's clear the relationship between the two? Either before or after I don't think it makes any difference.